source/speech.py

```diff
@@ -587,6 +586,10 @@ def speak(speechSequence,symbolLevel=None):
 				speechSequence[index]+=CHUNK_SEPARATOR
 	getSynth().speak(speechSequence)
 
+def speakSelectedText(text):

```

Comment:
Please add a doc string to this, something like:
```suggestion
def speakSelectedText(text):
	""" Helper method to speak the provided text with the word "selected" appended.
		Implemented using L{speakSelectionMessage}, which allows for speaking text with an arbitrary attached message.
	"""
```

Reply:
@feerrenrut Doc string added. I've assumed that it isn't necessary because many functions in this file don't have one, but of course having it makes code looks better. Thanks for your suggestion.